writers/mongodb/mongodb.go

```diff
@@ -0,0 +1,29 @@
+package mongodb
+
+import (
+	"gopkg.in/mgo.v2"
```

Comment:
While it will probably satisfy every requirement we have, this driver is [not anymore maintained](https://github.com/go-mgo/mgo). Please consider using (what seems to be) the [official driver](https://github.com/mongodb/mongo-go-driver).

Also, don't forget to update the platform dependencies!